src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp

```diff
@@ -55,8 +55,12 @@ namespace Opm {
         const TableContainer& sgfnTables = tm->getSgfnTables();
 
 
-        bool family1 = (!sgofTables.empty() || !slgofTables.empty()) && !swofTables.empty();
-        bool family2 = !swfnTables.empty() && !sgfnTables.empty() && !sof3Tables.empty();
+        bool family1 = !sgofTables.empty() || !swofTables.empty() || !slgofTables.empty();
+        bool family2 = !swfnTables.empty() || !sgfnTables.empty() || !sof3Tables.empty();
+
+        if (!swofTables.empty() && !slgofTables.empty()) {
```

Comment:
The diff in `SatfuncPropertyInitializers.cpp` seems unrelated to `THPRESFT`- PR pollution?

Reply:
that was unintentional, yes. shall I split it into a separate patch and leave it in on PR, do you want a separate PR, or shall it stay like it is in order to reduce the number of emails in people's inboxes?

Reply:
Assuming the offending changes should have been part of #692 I would like you to fixup that and we can merge that out of the way.

Reply:
okay, I factored this out into a separate patch and moved it to #692.